schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/dm/JasSubscription.java

```diff
@@ -7,6 +9,12 @@
 public class JasSubscription implements JasEndpointEntity {
     private String id;
 
+    private Date created;
+
+    private JasUser user;
```

Comment:
I think we could use just User here and google-endpoints would know to apply the userTransformer...